GaugeKit/Gauge.swift

```diff
@@ -87,6 +87,18 @@ public class Gauge: UIView {
             updateLayerProperties()
         }
     }
+    @IBInspectable public var topGradientColor: UIColor = UIColor.greenColor() {
+        didSet {
+            resetLayers()
+            updateLayerProperties()
+        }
+    }
+    @IBInspectable public var bottomGradientColor: UIColor = UIColor.redColor() {
```

Comment:
Are you sure, that we have to add 2 additional properties? Why you don't use `startColor` and `endColor` properties in `Gauge.swift`? they are useless in circle yet. I think too many properties can confuse users.